### PR TITLE
CW-person

### PR DESCRIPTION
**Pull request by aepaysinger**
this was a pretty basic one. let me know if you want me to create any methods or anything else on it

### CODE REVIEW

File: code_challenges/code_wars/person.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Person:
+    def __init__(self, first_name, last_name, age):
+        self.first_name = first_name
+        self.last_name = last_name
+        self.age = age
+        self.full_name = first_name + " " + last_name
```
Comment (W-Ely) [inline code_challenges/code_wars/person.py:6]: This is a-okay if you prefer. I almost always prefer f-strings like so `self.full_name = f"{first_name} {last_name}"`

**Review by W-Ely — APPROVED** [review]
Great!

File: README.md [hunk 2]
```diff
@@ -59,3 +59,8 @@
   * Tests: `tests/code_wars/test_trigrams.py`
   * URL: [challenge url](https://www.codewars.com/kata/55d8dc4c8e629e55dc000068/python)
   * Run time: (O)n
+### Person (7kyu)
+  * Module: `code_challenges/code_wars/person.py`
+  * Tests: `tests/code_wars/test_person.py`
+  * URL: [Challenge url](https://www.codewars.com/kata/513f887e484edf3eb3000001/python)
+  * Run Time: (O)1
```
[the remark this answers is not in the document]
Comment (aepaysinger) [reply README.md]: When its a situation like this and I make these changes do you want me to merge without waiting?

[the remark this answers is not in the document]
Comment (W-Ely) [reply README.md]: Yup, if I've approved and there is a small change to make.  Feel free to merge after making the change.

**Pull request merged**